app/api/monitoring/performance/route.ts

```
@@ -0,0 +1,253 @@
+            performanceMonitor.acknowledgeAlert(alertId)
+            return NextResponse.json({ success: true, message: 'Alert acknowledged' })
+          }
+          break
+
+        case 'getDetailedMetrics':
+          const detailedMetrics = {
+            performanceSummary: performanceMonitor.getPerformanceSummary(timeWindow || 300000),
+            tradingImpact: performanceMonitor.getTradingPerformanceImpact(timeWindow || 300000),
+            trends: performanceMonitor.getPerformanceTrends(timeWindow || 3600000),
+            healthStatus: performanceMonitor.getAPIHealthStatus()
+          }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)